pennylane/ops/channel.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the available built-in noisy
+quantum channels supported by PennyLane, as well as their conventions.
+"""
+# pylint:disable=abstract-method,arguments-differ,protected-access
+
+import numpy as np
+
+from pennylane.operation import AnyWires, Channel
+
+
+class QubitChannel(Channel):
+    r"""QubitChannel(K_list, wires)
+    Apply an arbitrary fixed quantum channel.
+
+    Kraus matrices that represent the fixed channel are provided
+    as a list of numpy array.
+
+    **Details:**
+
+    * Number of wires: Any (the operation can act on any number of wires)
+    * Number of parameters: 1
+    * Gradient recipe: None
+
+    Args:
+        K_list (list[array[complex]]): List of Kraus matrices
+        wires (Sequence[int] or int): the wire(s) the operation acts on
```

Comment:
Saw that this is identical to the docstring of other ops, would have to be updated 
```suggestion
        wires (Union[Wires, Sequence[int], or int]): the wire(s) the operation acts on
```